css-view-transitions-1/Overview.bs

```diff
@@ -1088,6 +1041,22 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 				1. Append |old| to |imagePair|.
 
+				1. Let |oldViewBox| be |capturedElement|'s [=old styles=] 'object-view-box' property.
+
+				1. Set |capturedElement|'s [=captured element/old view-box rule=] to a new {{CSSStyleRule}} representing the following CSS,
+					and append it to |document|'s [=document/view transition style sheet=].
+
+					<!-- deliberately using <pre> so the example can contain <var> references -->
+					<pre highlight="css">
+						html::view-transition-old(<var>transitionName</var>) {
+							object-view-box: <var>oldViewBox</var>;
+						}
+					</pre>
+
+					Note: The above code example contains variables to be replaced.
+
+					Issue: Which of ''xywh()''/''rect()''/''inset()'' should we use?
```

Comment:
Can this be resolved at editor's discretion?

Reply:
Any preferences?

Reply:
Settled